delocate/tools.py

```diff
@@ -377,6 +377,10 @@ def dir2zip(in_dir, zip_fname):
             # Preserve file permissions, but allow copy
             info = zipfile.ZipInfo(in_fname)
             info.filename = relpath(in_fname, in_dir)
+            if os.name == 'nt':
+                # Make the path unix friendly on windows.
+                # PyPI won't accept wheels with windows path separators
+                info.filename = relpath(in_fname, in_dir).replace("\\", "/")
```

Comment:
Just checking - but this should never have a drive letter in - right?

Reply:
Right. It's a relative path, so no drive letters.